libs/shared/ui/src/lib/components/error-boundary/error-boundary.tsx

```diff
@@ -0,0 +1,42 @@
+import { type PropsWithChildren } from 'react'
+import { ErrorBoundary as ErrorBoundaryRaw } from 'react-error-boundary'
+import { type FallbackProps } from 'react-error-boundary'
+import { useLocation } from 'react-router-dom'
```

Comment:
Duplicate import for `react-error-boundary`

Reply:
Fixed https://github.com/Qovery/console/pull/1403/commits/b57dc584729cbe16937b5db1f215935ff9bed56a